src/cmd-diff

```diff
@@ -114,6 +114,30 @@ def parse_args():
     return parser.parse_args()
 
 
+def diff_source_control(diff_from, diff_to):
```

Comment:
This is fine and clearly useful as is, but assuming that it's on GitHub seems like a leap for the developer case, no? It seems like it'd be cleaner to just do the git diff ourselves given that we have `src/config`.

I guess for the cosa case, the GitHub link is the best we can do.

Reply:
changed this behavior here to just output the URL if it is GitHub.